### PR TITLE
get_structure changed slightly

### PR DESCRIPTION
**Pull request by samwaseda**
This makes particularly sense for parent jobs if they should get information from child jobs instead of having an hdf5 file themselves.

### CODE REVIEW

**Comment by samwaseda** [comment]
Here's the [notebook](https://github.com/pyiron/pyiron/files/3466554/get_structure.ipynb.txt) that I used for testing

**Comment by jan-janssen** [comment]
Can you add the corresponding unit tests? or extend the existing unit tests for the testing job https://github.com/pyiron/pyiron/blob/master/tests/testing/test_exampleJob.py

File: pyiron/atomistics/job/atomistic.py [hunk 1]
```diff
@@ -508,14 +508,18 @@ def get_structure(self, iteration_step=-1, wrap_atoms=True):
...
             raise AssertionError()
         snapshot = self.structure.copy()
-        snapshot.cell = self.get("output/generic/cells")[iteration_step]
-        snapshot.positions = self.get("output/generic/positions")[iteration_step]
-        indices = self.get("output/generic/indices")
-        if indices is not None:
+        snapshot.cell = self.output.cells[iteration_step]
+        snapshot.positions = self.output.positions[iteration_step]
+        indices = self.output.indices
+        if indices is not None and len(indices)>max([iteration_step,0]):
```
Comment (sudarsan-surendralal) [inline pyiron/atomistics/job/atomistic.py]: Spacing for pep8 

`if indices is not None and len(indices) > max([iteration_step,0]):`

File: pyiron/atomistics/job/atomistic.py [hunk 2]
```diff
@@ -508,14 +508,18 @@ def get_structure(self, iteration_step=-1, wrap_atoms=True):
...
             snapshot.indices = indices[iteration_step]
         if wrap_atoms:
             return snapshot.center_coordinates_in_unit_cell()
         else:
+            if len(self.output.unwrapped_positions)>max([iteration_step,0]):
```
Comment (sudarsan-surendralal) [inline pyiron/atomistics/job/atomistic.py]: pep8 spacing (see above)

File: pyiron/atomistics/job/atomistic.py [hunk 3]
```diff
@@ -508,14 +508,18 @@ def get_structure(self, iteration_step=-1, wrap_atoms=True):
...
+                snapshot.positions = self.output.unwrapped_positions[iteration_step]
+            else:
+                snapshot.positions += self.output.total_displacements[iteration_step]
```
Comment (sudarsan-surendralal) [inline pyiron/atomistics/job/atomistic.py]: Is `self.output.total_displacements` implemented for all types of jobs? (LAMMPS, VASP and SPHINX)

Comment (samwaseda) [reply pyiron/atomistics/job/atomistic.py]: This comes from `GenericOutput` in `atomistic` so it should be there by default

**Comment by sudarsan-surendralal** [comment]
Merge with the master and then push! It was an error with the path (`../CHGCAR_samples` does not exist).

File: tests/testing/test_exampleJob.py [hunk 4]
```diff
@@ -56,6 +56,5 @@ def test_output(self):
             self.assertTrue(1 > e_1 > 0)
             self.assertEqual(e_1, e_2)
 
-
```
Comment (jan-janssen) [inline tests/testing/test_exampleJob.py]: PEP8 requires two empty lines here.

File: tests/testing/test_exampleAtomisticJob.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        job = project.load(project.get_job_ids()[0])
+        job.remove()
+        project.remove(enable=True)
+
+    def test_get_structure(self):
+        struct = self.job.get_structure()
+        self.assertIsNotNone(struct)
+        self.assertEqual(struct.positions.shape, (2, 3))
+        self.assertEqual(struct.cell.shape, (3, 3))
+
```
Comment (jan-janssen) [inline tests/testing/test_exampleAtomisticJob.py]: PEP8 again - two empty lines are required.

**Pull request merged**